src/encoded/static/scss/encoded/modules/_regulome_search.scss

```diff
@@ -1143,6 +1274,21 @@ button {
     .bar {
         margin-bottom: 0;
     }
+    .bar-annotation {
+        top: 4px;
+    }
+    .bar-container {
+        height: 24px;
+        @media screen and (max-width: $screen-sm-min) {
+            height: 28px;
+        }
+        .bar {
+            height: 24px;
+            @media screen and (max-width: $screen-sm-min) {
+                height: 28px;
+            }
+        }
+    }
```

Comment:
Interesting you always use max-width (desktop first) and I always use min-width (mobile first). At least in encoded, we might need to settle on something.

Reply:
OK! We should probably stick with min-width for Encoded since that's already the convention.